### ISSUE TITLE
Routing

### ISSUE THREAD

**Issue by ozydingo** [ozydingo/factory_burgers#2]
We're using

```ruby
mount FactoryBurgers::App.new, at: "/factory_burgers", as: "factory_burgers"
```

to serve the main static asset bundle. How do we get nested paths to respond differently?

-> Use `Rack::Builder`, and `map` the desired routes to other apps.

**Comment by ozydingo** [comment]
Try this out with a single nested route, then build out the rest.

**Comment by ozydingo** [comment]
Solved by 5f57ac9043b31497fa771ef75709d806712210cb
